cea/technologies/load_distribution.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
```

Comment:
it seems like this load distristribution method is specific to VCC.
is there a reason why it is a saperated script?

Reply:
No particular reason. I will transfer it into the VCC module once the functions are in physics.